### PR TITLE
Forgot Password Story

### PR DESCRIPTION
**Pull request by akbarb24**

### CODE REVIEW

**Comment by akbarb24** [comment]
You're right. I have moved to User service

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/controller/AuthController.java [hunk 1]
```diff
@@ -3,8 +3,10 @@
 import com.mitrais.chipper.temankondangan.backendapps.common.CommonResource;
 import com.mitrais.chipper.temankondangan.backendapps.common.response.ResponseBody;
 import com.mitrais.chipper.temankondangan.backendapps.model.User;
+import com.mitrais.chipper.temankondangan.backendapps.model.json.ForgotPasswordWrapper;
```
Comment (donikrizky) [inline src/main/java/com/mitrais/chipper/temankondangan/backendapps/controller/AuthController.java:6]: there is still unused imported package here

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/controller/AuthController.java [hunk 2]
```diff
@@ -3,8 +3,10 @@
...
 import com.mitrais.chipper.temankondangan.backendapps.model.json.LoginWrapper;
 import com.mitrais.chipper.temankondangan.backendapps.model.json.RegisterUserWrapper;
+import com.mitrais.chipper.temankondangan.backendapps.model.json.ResetPasswordWrapper;
```
Comment (donikrizky) [inline src/main/java/com/mitrais/chipper/temankondangan/backendapps/controller/AuthController.java:9]: there is still unused imported package here

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/AuthService.java [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 import com.mitrais.chipper.temankondangan.backendapps.model.User;
 import com.mitrais.chipper.temankondangan.backendapps.model.json.RegisterUserWrapper;
+import com.mitrais.chipper.temankondangan.backendapps.model.json.ResetPasswordWrapper;
```
Comment (donikrizky) [inline src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/AuthService.java:5]: there is still unused imported package here

File: src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/AuthServiceImpl.java [hunk 4]
```diff
@@ -1,40 +1,54 @@
...
 public class AuthServiceImpl implements AuthService {
+    private final static Logger logger = LoggerFactory.getLogger(AuthServiceImpl.class);
 
-    @Autowired
     PasswordEncoder passwordEncoder;
-
-    @Autowired
     UserRepository userRepository;
+    ProfileRepository profileRepository;
+    SimpleMailMessage template;
```
Comment (donikrizky) [inline src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/AuthServiceImpl.java:44]: is SimpleMailMessage still used here?

**Pull request merged**